src/sql/workbench/services/dialog/common/dialogTypes.ts

```diff
@@ -329,4 +366,26 @@ export class Wizard {
 		this._message = value;
 		this._onMessageChange.fire(this._message);
 	}
+
+	public get loading(): boolean {
+		return this._loading;
+	}
+
+	public set loading(value: boolean) {
+		if (this.loading !== value) {
+			this._loading = value;
+			this._onLoadingChange.fire({
+				loadingState: this._loading,
+				loadingMessage: this._loadingMessage
+			});
+		}
+	}
+
+	public get loadingMessage(): string | undefined {
+		return this._loadingMessage;
+	}
+
+	public set loadingMessage(value: string | undefined) {
```

Comment:
We generally should also be having a loading completed message

And both the loading messages should generally have some good default message like we do in the loading component.